Make info use optional extra parameter

PanMan opened the pull request:
The Redis Info call accepts a second parameter to only return part of the info. E.g. Info Memory. This pull request should make the library accept that.

PanMan committed:
Make info use optional extra parameter

Info accepts a second parameter to only return part of the info. E.g. Info Memory. This commit should make the library accept that.

joelcox commented:
Hey PanManAms,

Thanks for your pull request. Would you mind updating the documentation and write a test for this new functionality? I'd also appreciate it if you updated to code to reflect the style specified in the style guide.

Thanks!

On Jun 17, 2013, at 3:39 PM, PanManAms notifications@github.com wrote:

> The Redis Info call accepts a second parameter to only return part of the info. E.g. Info Memory. This pull request should make the library accept that.
> 
> You can merge this Pull Request by running
> 
>   git pull https://github.com/PanManAms/codeigniter-redis develop
> Or view, comment on, or merge it at:
> 
>   https://github.com/joelcox/codeigniter-redis/pull/37
> 
> Commit Summary
> 
> Make info use optional extra parameter
> File Changes
> 
> M libraries/Redis.php (8)
> Patch Links:
> 
> https://github.com/joelcox/codeigniter-redis/pull/37.patch
> https://github.com/joelcox/codeigniter-redis/pull/37.diff

PanMan committed:
Created test for info with specific section

Added test_info_section which should get only the memory section of the info command

PanMan committed:
Changed coding style, variable name

PanMan committed:
Changed false to FALSE as styleguide mandates

PanMan commented:
I have created a test and changed the coding style. I assume you mean the CodeIgniter style guide? I think the command doesn't need any special documentation, as none of the commands are explained in the readme, and it exactly follows the doc at http://redis.io/commands/info

joelcox commented:
Thanks for your contribution :)
